tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh

```diff
@@ -53,19 +53,21 @@ function install_packages() {
   fi
 
   ${apt_get} install grub-legacy \
-    rsyslog logrotate cron insserv net-tools ifupdown vim tmux netbase iptables \
+    rsyslog logrotate cron net-tools ifupdown vim tmux netbase iptables \
     openssh-server e2fsprogs tcpdump socat wget \
     python bzip2 sed gawk diffutils grep gzip less tar telnet ftp rsync traceroute psmisc lsof procps \
     inetutils-ping iputils-arping httping  curl \
-    dnsutils zip unzip ethtool uuid file iproute acpid virt-what sudo \
+    dnsutils zip unzip ethtool uuid file iproute acpid sudo \
     sysstat python-netaddr \
     apache2 ssl-cert \
     dnsmasq dnsmasq-utils \
     nfs-common \
     samba-common cifs-utils \
     xl2tpd bcrelay ppp ipsec-tools tdb-tools \
     xenstore-utils libxenstore3.0 \
-    conntrackd ipvsadm libnetfilter-conntrack3 libnl-3-200 libnl-genl-3-200 \
+    ipvsadm conntrackd libnetfilter-conntrack3 \
+    keepalived irqbalance \
+    libnl-3-200 libnl-genl-3-200 \
```

Comment:
I doubt that these 2 libs have to be explicitly installed. They may get installed as dependency though. Would you mind to take them out here?

Reply:
Sure, I'll check and remove them from the list if they can be removed.